x-pack/plugins/synthetics/server/synthetics_service/project_monitor_formatter.ts

```diff
@@ -105,13 +105,10 @@ export class ProjectMonitorFormatter {
 
   public configureAllProjectMonitors = async () => {
     this.staleMonitorsMap = await this.getAllProjectMonitorsForProject();
-    await Promise.all(
-      this.monitors.map((monitor) =>
-        this.configureProjectMonitor({
-          monitor,
-        })
-      )
-    );
+    for (const monitor of this.monitors) {
+      await this.configureProjectMonitor({ monitor });
+      await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
What is this for?

Reply:
without some delay, this wasn't working. Not sure why. But that's the case.

Reply:
In a small follow up, we can try to separate code paths for public/private locations.